config/default.go

```diff
@@ -34,14 +34,16 @@ const DefaultValues = `
 	MaxConns = 200
 
 [EthTxManager]
-	FrequencyToMonitorTxs = "1s"
-	WaitTxToBeMined = "2m"
-	ForcedGas = 0
-	GasPriceMarginFactor = 1
-	MaxGasPriceLimit = 0
-	PrivateKeys = [
-		{Path = "/pk/agglayer.keystore", Password = "testonly"},
-	]
+	[EthTxManager.Base]
```

Comment:
Why not everything at the base level?

Reply:
Fixed https://github.com/0xPolygon/agglayer/pull/79/commits/f1fe0ead9aa41baf3e62c156e4457447d74f0567